axelar-chains-config/info/devnet-amplifier.json

```
@@ -1322,6 +1322,135 @@
           "version": "2.1.1"
         }
       }
+    },
+    "stacks": {
+      "name": "Stacks",
+      "axelarId": "stacks",
+      "rpc": "https://api.testnet.hiro.so",
+      "tokenSymbol": "STX",
+      "networkType": "testnet",
+      "chainType": "stacks",
+      "finality": "1",
+      "decimals": 6,
+      "approxFinalityWaitTime": 1,
+      "explorer": {
+        "name": "Stacks Explorer",
+        "url": "https://explorer.hiro.so/?chain=testnet"
+      },
+      "contracts": {
+        "ClarityStacks": {
+          "address": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1.clarity-stacks",
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1"
+        },
+        "GasStorage": {
+          "address": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1.gas-storage",
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1"
+        },
+        "Traits": {
+          "address": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1.traits",
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1"
+        },
+        "GasImpl": {
+          "address": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1.gas-impl",
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1",
+          "version": "1.3.0"
+        },
+        "GatewayStorage": {
+          "address": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1.gateway-storage",
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1"
+        },
+        "GasService": {
+          "address": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1.gas-service",
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1",
+          "gasCollector": "ST3ASWHNH5G542F1V58CQAND5GFN4ZDGXAJHC2VR0"
+        },
+        "GatewayImpl": {
+          "address": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1.gateway-impl",
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1",
+          "version": "1.3.0"
+        },
+        "Gateway": {
```

```suggestion
        "AxelarGateway": {
```
can you preserve the naming in the configs similar to other chains for consistency?